src/GameController/Enums.cs

```diff
@@ -12,6 +12,9 @@
 using Foundation;
 
 namespace GameController {
+	[Deprecated (PlatformName.MacOSX, 10, 15, message: "GCMicroGamepadSnapshot has been deprecated, use [GCController controllerWithMicroGamepad] instead")]
```

Comment:
In deprecation messages that point elsewhere, we point to the managed name instead of the Objective-C name, since that's much more useful for our developers.

Reply:
Ah man looked right past that